src/shared/SellerProfile/ReviewHeader.js

```diff
@@ -19,9 +19,8 @@ const ReviewHeader = (props) => {
   } = props;
 
   const badgeTitle = {
-    indigenous: 'Indigenous',
-    nfp_social_enterprise: 'Not-for-profit / social enterprise',
-    product: 'Product based business',
```

Comment:
For now yea, I guess we could temporarily leave the badge. It will inevitably disappear though.